src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -419,6 +419,37 @@ protected final void execute(Configuration config, String... filenames) throws E
         checker.destroy();
     }
 
+    /**
+     * Executes given config on a list of files only. Does not verify violations.
+     *
+     * @param moduleConfig check configuration
+     * @param fileName names of files to process
+     * @param expectedViolations a map of expected violations per files.
+     * @param suppressedViolations suppressed violation messages
+     * @throws Exception if there is a problem during checker configuration
+     */
+    protected final void execute(Configuration moduleConfig, String fileName,
+                                 String[] expectedViolations, String... suppressedViolations)
```

Comment:
We do not need these parameters

Reply:
https://github.com/checkstyle/checkstyle/commit/e77a62f2bffc200f9eb5b6702e6d3d3fc50d8ce8

Reply:
https://github.com/checkstyle/checkstyle/pull/11465/files

Reply:
But we have the `verify `method in `verifySuppressed` which uses these 2 parameters like `expected violation` and `suppressed violations`

Reply:
https://github.com/checkstyle/checkstyle/commit/e77a62f2bffc200f9eb5b6702e6d3d3fc50d8ce8#diff-cf838e5bbdb7de5a3e0e91f9919a389a0fd2ffdbda60c85a3c6dba07dcf69acdR406-R420 only deals with config and file name and when I am using this execute method then we can't use expected and suppressed violation

Reply:
You need to make verifySuppressed that takes as argument only config and input file.
Such method should run "execute" method.

We do not care about any violation and any suppression, we expect exception before file even parsed

Reply:
```
    private void verifySuppressed(Configuration moduleConfig, String fileName)
        throws Exception {

        execute(moduleConfig, getPath(fileName));
    }
```

Like this?

Reply:
Yes

Reply:
New method should be created.

Please do not push your code until `mvn verify ` is passing on local.

Reply:
I have created the method but where should I use it?

Because I can't use it in :
```
        verify(checkerConfig, fileName,
               removeSuppressed(expectedViolations, suppressedViolations));
```

Reply:
Need Help: 

1. I started with creating:     
```
private void verifySuppressed(Configuration moduleConfig, String fileName)
        throws Exception {

        execute(moduleConfig, getPath(fileName));
    }
``` 
in the ` SuppressionCommentFilterTest.java`


2.  I saw that I should change this `verifySuppressed `method of 3 parameters with my `verifySuppressed `method that only takes 2 parameters.

```
    @Test
    public void testInvalidMessageFormat() throws Exception {
        final DefaultConfiguration filterConfig =
            createModuleConfig(SuppressionCommentFilter.class);
        filterConfig.addProperty("messageFormat", "e[l");

        try {
            final String[] suppressed = CommonUtil.EMPTY_STRING_ARRAY;
            verifySuppressed(filterConfig, "InputSuppressionCommentFilter11.java", suppressed);
            assertWithMessage("Exception is expected").fail();
        }
        catch (CheckstyleException ex) {
            final IllegalArgumentException cause = (IllegalArgumentException) ex.getCause();
            assertWithMessage("Invalid exception message")
                .that(cause)
                .hasMessageThat()
                .isEqualTo("unable to parse expanded comment e[l");
        }
    }
```

Like this --> 

```
        try {
            verifySuppressed(filterConfig, "InputSuppressionCommentFilter11.java");
            assertWithMessage("Exception is expected").fail();
        }
```

But now I am getting an error of **`EXCEPTION IS EXPECTED`**

PS; I haven't changed the verify() method in `verifySuppressed method` which takes 4 parameter

Reply:
you updating too much.

you need to remove methods:
```
-    private void verifySuppressed(Configuration moduleConfig, String fileName,
-            String... aSuppressed)
-            throws Exception {
- 
-    private void verifySuppressed(Configuration moduleConfig, String fileName,
-            String[] expectedViolations, String... suppressedViolations) throws Exception {
```

and figure out how to upadate test that that used them to execute validation by checkstyle but wihtout comparison with expected violation, as you expect exception.

test has problem with path to file and Input file is over complicated for no reason, so please redo Input file content  and have only config for filter and for one Check and in code to have only one violation (no more code in Input file is requied).

time to debug how Checks are working :).